.github/dependabot.yml

```
@@ -24,9 +26,10 @@ updates:
   - package-ecosystem: "cargo"
     directory: "/"
     schedule:
-      interval: "weekly"
-      day: "monday"
+      interval: "monthly"
       time: "00:00"
+    cooldown:
+      default-days: 5
```

In light of [STR-1827](https://alpenlabs.atlassian.net/browse/STR-1827) we should change this to

```suggestion
    cooldown:
      default-days: 14
```